agent/logging.go

```diff
@@ -172,31 +171,8 @@ func (a *Agent) prepSingleLogger(tc *taskContext, in model.LogOpts, logDir, file
 	}
 }
 
-func redactList(projectVars map[string]string, redacted map[string]bool) []string {
-	suspiciousPatterns := []string{
-		"key",
-		"secret",
-		"auth",
-		"token",
-		"private",
-		"pass",
-		"pw",
-	}
-
+func redactList(redacted map[string]bool) []string {
```

Comment:
tiny nit: redactList is a bit confusing since redact is a verb, but we aren't actually doing a redaction here, we're just getting the list of redacted keys (this took me a minute to figure out). Could we rename to getAllRedactedKeys or something?

Reply:
and/or throw a comment on

Reply:
Done